toolkit.lisp

```diff
@@ -266,6 +266,7 @@
     ((or string pathname)
      (open designator :direction direction
                       :if-exists if-exists
+                      :if-does-not-exist :create
```

Comment:
This is wrong. `:if-does-not-exist` should not be `:create` if the direction is `:input`. Furthermore, CLHS specifies that `:if-does-not-exist` must already be `:create` if the direction is `:output` anyway.